src/core/utils/components/VirtualScroll.tsx

```diff
@@ -42,7 +53,11 @@ const getNumberOfNodesInHeight = (childHeight: number, totalHeight: number) => {
 };
 
 const getTranslateValue = (childHeight: number, startIndex: number) => {
-  return childHeight * startIndex;
+  if (startIndex > 0) {
```

Comment:
I couldn't find a case where startIndex isn't greater than 0. Was this `if` statement added just for clarification?

Reply:
Wont be able to remember, to be honest.. but might have been some edge case or so :/